vignettes/adrs.Rmd

```diff
@@ -4,57 +4,62 @@ output:
   rmarkdown::html_vignette
 vignette: >
   %\VignetteIndexEntry{Creating ADRS}
-  %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: 72
```

Comment:
What does this option do?

Reply:
I am not seeing a difference with/without it, so I've removed.